src/content/docs/infrastructure/host-integrations/host-integrations-list/nvidia-jetson-integration.mdx

```diff
@@ -0,0 +1,164 @@
+---
+title: NVIDIA Jetson integration
+tags:
+    - New Relic integrations
+    - NVIDIA Jetson integration
+metaDescription: Use New Relic infrastructure agent to get a dashboard with metrics from your NVIDIA Jetson.
+freshnessValidatedDate: 2024-06-18
+---
+import jetsonDashboard from 'images/infrastructure_screenshot_full-nvidia-jetson-dashboard.webp'
+
+
+Our NVIDIA Jetson integration collects and sends data about your tegrastats utility to New Relic, where you can monitor CPU, GPU, RAM, and temperature readings about your NVIDIA Jetson device. 
```

Comment:
@sjyothi54 I looked up NVIDIA Jetson and saw that it's both a hardware device and may also include some software. It seems like this integration is mostly concerned with the hardware side of the device... Is that accurate?

Reply:
@akristen Yes, Jetson is a hardware device. Users of Jetson can obtain metrics, such as tegrastats, by default. By using our nri-flex integration, they can seamlessly send these metrics to New Relic.